README.md

```diff
@@ -89,9 +90,17 @@ This command will run the example test based on the `./.sauce/config.yml` file.
 ```sh
 saucectl run --timeout <seconds>
 ```
-using the --timeout flag will set the test timeout for the test runner framework. 
+Using the `--timeout` flag will set the test timeout for the test runner framework. 
 
 
+#### Sauce Labs Region
```

Comment:
Here is where I would just say just add one paragraph saying: `saucectl` supports various command line parameter to modify the test, check them out in the saucectl project (and add a link to the saucectl docs).

Reply:
👍 lets do that now then :) I'll update the saucectl readme as well.